kornia/augmentation/base.py

```diff
@@ -341,28 +343,29 @@ def inverse_transform(
         input: torch.Tensor,
         transform: Optional[torch.Tensor] = None,
         size: Optional[Tuple[int, int]] = None,
-        **kwargs
+        **kwargs,
     ) -> torch.Tensor:
-        """By default, the exact transformation as ``apply_transform`` will be used.
-        """
+        """By default, the exact transformation as ``apply_transform`` will be used."""
         return self.apply_transform(input, params=self._params, transform=transform)
 
     def compute_inverse_transformation(self, transform: torch.Tensor):
-        """Compute the inverse transform of given transformation matrices.
-        """
+        """Compute the inverse transform of given transformation matrices."""
         return _torch_inverse_cast(transform)
 
     def inverse(
         self,
         input: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]],
         params: Optional[Dict[str, torch.Tensor]] = None,
         size: Optional[Tuple[int, int]] = None,
-        **kwargs
+        **kwargs,
     ) -> torch.Tensor:
-        if isinstance(input, (
-            list,
-            tuple,
-        )):
+        if isinstance(
+            input,
+            (
+                list,
+                tuple,
+            ),
+        ):
```

Comment:
this is formatting I dont like about Black :D

Reply:
This is because the line has a "magic comma" aka "trailing comma" so black expands it.

Ideally, black would evaluate whether the expression can be written in a single line given the maximum line length (even if it has a trailing comma) and then it would choose whether to expand it or not.
But I don't think black does that.

@asottile First of all, you are a legend, big fan of your work!
I was wondering if https://github.com/asottile/add-trailing-comma could be expanded to always remove trailing commas.

This way we could run your tool, and then black - thus making sure that all trialing commas are there because the expression wouldn't fit otherwise

Reply:
so `add-trailing-comma` already removes unnecessary inline commas -- so maybe it just needs to be run once before black?

Reply:
p.s. I also don't really personally like black, but often it's the "easiest" choice

Reply:
> so add-trailing-comma already removes unnecessary inline commas -- so maybe it just needs to be run once before black?

Only does it if in a single line:

```diff
-if isinstance(input, (list, tuple,),):
+if isinstance(input, (list, tuple)):
     pass
```

But using the case above, this conversion does not happen:
```diff
 if isinstance(
     input,
     (
        list,
-       tuple,
+       tuple
-    ),
+    )
 ):
     pass
```

If it did, we could then run `black` and get what we want:

``` python
if isinstance(input, (list, tuple)):
    pass
```

Since the `isinstance(...)` check is well under the line length limit if on one line.

So what I would like is a `remove-trailing-comma` tool that can be run before `black`
This way black is forced to re-evaluate whether to expand expressions or not.

Do you prefer if we move this discussion elsewhere?

Reply:
@carmocca I found reverse https://github.com/asottile/add-trailing-comma

Reply:
@carmocca yeah -- what I'm suggesting is to put `add-trailing-comma` before `black` inside the `.pre-commit-config.yaml` -- this would ensure that inline extra-commas are removed before black messes them up

Reply:
even better -- there's a `--skip-magic-trailing-comma` option to `black` -- ezpz!

let me recreate this PR using that option

Reply:
done!